errors/errors.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2016 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package errors
```

Comment:
wonder if you don't want to rename it avoid clashing everywhere, like actually `merror` ?

Reply:
yeah I thought about that, but in practise I think that more things will get added  to the tsdb/errors package so at one point it will become more than just `merror`

Reply:
[YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) mate (: 

Just don't predict future and make your life easier now (:

Reply:
nice I try to follow the same principle in general.

interested in ine more opinion and will rename.

Reply:
it seems goutham and ganesh are busy on some other fronts so merging as is to unblock another PR that depends on this change and will rename later if/when needed.